hack/upgrade-test-operator-sdk.sh

```diff
@@ -0,0 +1,315 @@
+#!/bin/bash -e
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2021 Red Hat, Inc.
+#
+# Usage:
+# make upgrade-test-index-image
+#
+# Use Openshift-CI "optional-operators-ci-*" workflow to:
+# - Build an internal index image based off of the index image at:
+#   quay.io/kubevirt/hyperconverged-cluster-index
+#   with the appropriate tag of the version
+# - Add to that index a new bundle, named 100.0.0 with the contents
+#   of the open PR (this can include new dependent images, new CRDs...).
+# - Subscribe to the initial channel, using the "optional-operators-ci-subscribe"
+#   step.
+# This script is then upgrading HCO to 100.0.0 version, by patching the subscripiton,
+# and performs various validations against the upgraded version.
```

Comment:
i think this description should be updated, according to the new method with operator-sdk tool